src/Security/Authentication/OpenIdConnect/src/OpenIdConnectHandler.cs

```diff
@@ -105,7 +105,7 @@ protected virtual async Task<bool> HandleRemoteSignOutAsync()
               && Request.ContentType.StartsWith("application/x-www-form-urlencoded", StringComparison.OrdinalIgnoreCase)
               && Request.Body.CanRead)
             {
-                var form = await Request.ReadFormAsync();
+                var form = await Request.ReadFormAsync(Context.RequestAborted);
```

Comment:
Why do we need this? This happens automagically without passing in the token.

Reply:
TIL - I didn't realise this was equivalent to the default behaviour with `CancellationToken.None`/`default`.